.storybook/util.js

```diff
@@ -3,7 +3,7 @@ const GITHUB_REPO = "https://github.com/wzulfikar/unwind"
 export const noop =
   (name) =>
     (...args) =>
-      console.log(name, args)
+      console.log(name, ...args)
```

Comment:
In before, the args will show up as array (eg. `Array(2)`) which is not that handy. Changing to `...args` makes it show the actual argument (so I don't need to click to see it).

| Before | After |
|--------|-------|
|       ![image](https://user-images.githubusercontent.com/7823011/206306588-3ebbc149-d375-4a4e-b5cf-44f45946d827.png) |     ![image](https://user-images.githubusercontent.com/7823011/206306616-0fdc3cd8-5fc5-4ddc-bf06-208220cac523.png)  |